protocols.csv

```diff
@@ -10,4 +10,5 @@ code	size	name
 421	V	ipfs
 480	0	http
 443	0	https
-444	10	onion
\ No newline at end of file
+477	0	websockets
```

Comment:
> From what I heard from @whyrusleeping, @jbenet proposed that ws means 'no path' and wsn represents a websockets addr with paths.

wait, i thought it was:

- `ws` is path without a number telling you long the path should be (`/ws/foo/bar/baz`) -- cannot encapsulate
- `wsn` is a path with a number telling the number of path components (`/ws/3/foo/bar/baz`) -- can encapsulate

we could also do:
- `ws16` a base16 (hex) encoded path or something `/ws/2f666f6f2f6261722f62617a0a` or `ws32` or `ws58` -- can encapsulate, variable length but not variable components